extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -146,62 +146,47 @@ describe.configure().retryOnSaucelabs().run('amp-bind', function() {
     });
   });
 
+  /**
+   * Combined test specs for amp-img
+   */
   describe('amp-img integration', () => {
-    it('should change src when the src attribute binding changes', () => {
-      const button = fixture.doc.getElementById('changeImgSrcButton');
+    it('should do amp-img integration', () => {
+      const changeImgSrcButton = fixture.doc
+          .getElementById('changeImgSrcButton');
+      const invalidSrcButton = fixture.doc.getElementById('invalidSrcButton');
+      const ftpSrcButton = fixture.doc.getElementById('ftpSrcButton');
+      const changeImgAltButton = fixture.doc
+          .getElementById('changeImgAltButton');
+      const changeImgDimensButton = fixture.doc
+          .getElementById('changeImgDimensButton');
       const img = fixture.doc.getElementById('image');
       expect(img.getAttribute('src')).to.equal('http://www.google.com/image1');
-      button.click();
+      expect(img.getAttribute('height')).to.equal('200');
+      expect(img.getAttribute('width')).to.equal('200');
+      changeImgSrcButton.click();
```

Comment:
I am not sure I got what you said. I think there is another expect that checks the image dimensions after the button is clicked. Initially, we expect to have the image in a specific dimension and then the dimension changes after the button is clicked (this is what I understood; please correct me if I'm wrong). 

`expect(img.getAttribute('height')).to.equal('200');
  expect(img.getAttribute('width')).to.equal('200');
  changeImgDimensButton.click();
  // Should change width and height when their bindings change
  expect(img.getAttribute('height')).to.equal('300');
  expect(img.getAttribute('width')).to.equal('300');`